src/common/darktable.c

```diff
@@ -1591,54 +1654,24 @@ void dt_configure_performance()
 
   fprintf(stderr, "[defaults] found a %zu-bit system with %zu kb ram and %zu cores (%d atom based)\n",
           bits, mem, threads, atom_cores);
-  if(mem >= (16lu << 20) && threads > 4)
+  if(mem >= (4lu << 20) && threads >= 2)
```

Comment:
This section is now quite minimal, either we set the perf to default or mini. Can't we have a better automatic setting based on the memory and cores as before?

Maybe keep this condition and set to `large`.

Reply:
default has been chosen because it is enough for most images and current machines and leaves enough headroom and host_memory to avoid fallbacks even if the user has other apps or a "hungry" desktop. This is basically all about a) "when to tile" and b) "avoid wrong don't tile" so default should be fine and should be set to large or others only by active user decision.

Reply:
Ok, understood!